tests/pages/adminPage.js

```diff
@@ -6,11 +6,11 @@ module.exports = {
     // setting locators
     url: "graph/d/Fxvd1timk/home-dashboard?orgId=1",
     fields: {
-        navigation: "//div[@class='navbar']//a",
-        timePickerMenu: "//button[@ng-click='ctrl.openDropdown()']",
+        navigation: "//div[@class='navbar']//a[2]", // /html/body/grafana-app/div/div/div/react-container/div/div[1]/div[1]/div/a[2]
+        timePickerMenu: "/html/body/grafana-app/div/div/div/react-container/div/div[1]/div[5]/div[1]/gf-time-picker/button[1]",// //button[@ng-click='ctrl.openDropdown()
```

Comment:
This Xpath is not efficient, and also not readable, please think of a better locator. Look at the ones I have used.